### PR TITLE
Design updates

### PR DESCRIPTION
**Pull request by nein09**

### CODE REVIEW

File: world/templates/geek_box.html [hunk 1]
```diff
@@ -1,14 +1,20 @@
...
-        <p>Have questions? Click <a href="mailto:{{ settings.contact_email }}">here</a> to email the maintainers of this specific site.</p>
-        <p>Mailing list signup and social media links go here?</p>
+      <div class="small-12 medium-6 columns">
+        <h6>How does it work?</h6>
+        <p>MissoulaReady is designed to help people prepare for disasters that occur in their area. Disasters don’t strike locations equally so we found it important to give people location
+        specific information in order to properly prepare. MissoulaReady organizes current information and packages it in a way that makes it accessible for any Missoula County
+        resident.</p>
+        <p>This site uses the most up-to-date hazard risk data available for Missoula County, Montana. The descriptions of risk and how to prepare for those risks are based on best information from the American Red Cross and the Federal Emergency Management Agency. Data used for MissoulaReady is available for download here.</p>
```
Comment (eldang) [inline world/templates/geek_box.html]: Why not link to github / world/data.zip here?

File: world/templates/geek_box.html [hunk 2]
```diff
@@ -1,14 +1,20 @@
...
+        <p>Have questions? <a href="mailto:{{ settings.contact_email }}">Email the creators</a> for more information.</p>
     </div>
     <div class="small-12 medium-6 columns">
-        <h6>Is this available for other locations?</h6>
+        <h6>Is this available for other Counties or States?</h6>
         <p>You can make it available! If you can put up a webserver and you have the data, you can set this up. Visit us on <a href="https://github.com/missoula-ready/disaster-preparedness">Github</a> for more information.</p>
     </div>
+    <div class="small-12 medium-6 columns">
+        <h6>Who made this?</h6>
+        <p>This is based on <a href="http://www.opb.org/news/widget/aftershock-find-your-cascadia-earthquake-story/">Aftershock</a>, an earthquake preparedness application for Oregon residents. Carson MacPherson-Krutsky and <a href="http://www.hs.umt.edu/geosciences/faculty/bendick/">Dr. Rebecca Bendick</a>, a graduate student and her advisor at the Unversity of Montana, had the idea to expand it for other locales and types of disasters. Melinda Minch and Eldan Goldenberg adapted it for that purpose.</p>
```
Comment (eldang) [inline world/templates/geek_box.html]: Let's put links in for ourselves - github profiles if nothing else.

File: world/templates/index.html [hunk 3]
```diff
@@ -29,16 +29,16 @@
...
     </div>
     <div class="information-instructions">
-      <p class="organization-information">{{ settings.about_text }} This site is designed to help people prepare for disasters that can occur in their area.</p>
+      <p class="organization-information">{{ settings.about_text }}</p>
       <p>{{ location.disaster_description }}</p>
-      <p class="location-instructions-large">Give your location by typing in the box below, clicking on the map, or clicking 'find me'. You'll get a personalized report on your risks and how to prepare.</p>
-      <p class="location-instructions-small">Give your location by typing in the box below or clicking 'find me'. You'll get a personalized report on your risks and how to prepare.</p>
+      <p class="location-instructions-large">Give your location by typing in the box below, clicking on the map, or clicking 'find me'. You'll get a personalized report on your risks and steps you can take to prepare.</p>
+      <p class="location-instructions-small">Give your location by typing in the box below or clicking 'find me'. You'll get a personalized report on your risks and steps you can take to prepare.</p>
```
Comment (eldang) [inline world/templates/index.html:41]: Does only one of these paragraphs show on any given client?  (If so, ignore this comment)

**Comment by eldang** [comment]
Minor comments - everything else looks good.

File: world/templates/geek_box.html (hunk 1 above)
Comment (nein09) [inline world/templates/geek_box.html]: oh yeah! I was wondering where that was going to go. And it's all available right here on teh internet.

File: world/templates/index.html (hunk 3 above)
Comment (nein09) [inline world/templates/index.html:41]: Yes! In the css, you can see them shown and hidden in the media queries.

**Pull request merged**